index.html

```diff
@@ -1,312 +1,401 @@
+                  class="skill_item_icon"
+                  alt="CSS Icon"
+                />
+                <p class="skills_name upper">css</p>
+              </li>
+            </ul>
           </li>
+
           <li>
-            <textarea class="text-message" name="message" id="message" rows="5" maxlength="500"
-              placeholder="Write your message here" required></textarea>
+            <ul class="skill_frame separator">
+              <li class="skills_title">Frameworks</li>
+              <li class="btn xs-transparent_btn btn_flex expand_collapse">
+                <i class="material-icons">chevron_right</i>
+              </li>
+            </ul>
           </li>
-          <li>
 
-            <button type="submit" class="btn btn_normal submit-button enabled">Get in Touch</button>
+          <li>
+            <ul class="skill_frame separator">
+              <li class="skills_title">Skills</li>
+              <li class="btn xs-transparent_btn btn_flex expand_collapse">
+                <i class="material-icons">chevron_right</i>
+              </li>
+            </ul>
           </li>
         </ul>
-      </form>
-      <div class="bottom-separator">&nbsp;</div>
-    </section>
-  </div>
-</body>
-
+      </section>
+
+      <section id="Contact" class="contact-me_section contact-me_sect_bg">
+        <div class="content_wrapper">
+          <h2 class="main_title fw_bold contact_title white">Contact me</h2>
+          <p class="contact_txt">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+
+          <form action="https://formspree.io/f/mdoyzdon" method="POST">
+            <ul class="contact_form">
+              <li>
+                <input
+                  class="bg_white"
+                  name="name"
+                  type="text"
+                  maxlength="30"
+                  placeholder="Write your name here"
+                  required
+                />
+              </li>
+              <li>
+                <input
+                  class="bg_white"
+                  name="email"
+                  type="email"
+                  placeholder="your_email@mail.com"
+                  required
+                />
+              </li>
+              <li>
+                <textarea
+                  class="text-message bg_white"
+                  name="message"
+                  id="message"
+                  rows="5"
+                  maxlength="500"
+                  placeholder="Write your message here"
+                  required
+                ></textarea>
+              </li>
+              <li class="submit_btn">
+                <button type="submit" class="btn action_btn enabled">
+                  Get in Touch
+                </button>
+              </li>
+            </ul>
+          </form>
```

Comment:
Kindly Increase the width of the form as shown below the difference between the original design and the current project

#### Original design
![right](https://user-images.githubusercontent.com/58376020/122479226-fffc4b80-cfca-11eb-8471-e101b592b560.png)

### Current project
![wrong1](https://user-images.githubusercontent.com/58376020/122479279-13a7b200-cfcb-11eb-9a11-12b1938e07b0.png)

Reply:
Hi @Shaher-11, thank you for taking the time to review my code. Inspecting my project page I noticed that the computed width is exactly the same size as the Figma template size. I attached a screenshot from the project preview and a screenshot from the template to compare.
![Figma_template@1440](https://user-images.githubusercontent.com/81151384/122532856-38c70f80-d021-11eb-80f2-5152469792d7.jpg)
![Portfolio_Form_preview@1440w](https://user-images.githubusercontent.com/81151384/122532983-5a27fb80-d021-11eb-8b4b-259d6c0b101f.jpg)